### ISSUE TITLE
setOptions({ global.useUTC: false }) causes problems with points positions.

### ISSUE THREAD

**Issue by Denyllon** [highcharts/highcharts#7257]
#### Expected behaviour

Points should be positioned in straight vertical line, in case of hovering points with the same timestamp values.

#### Actual behaviour

Points aren't placed in straight line.

#### Live demo with steps to reproduce

[https://jsfiddle.net/daniel_s/m369ea8t/](https://jsfiddle.net/daniel_s/m369ea8t/)
1. Change value of useUTC in global settings from **false** to **true**

#### Affected browser(s)

All browsers

**Comment by pawelfus** [comment]
*Internal note:*
Regression after 2d27725

**Comment by razvan-moldovan** [comment]
I found a similar behavior with useUTC, not sure it's related to this issue, but before submiting a new issue, I want to ask is this is related. Here is an reproduction of the issue I found:
[](https://jsfiddle.net/mrazvan/ny98jw2n/)
If you look at xAxis, which is a datetime type you'll see that 29th October is duplicated but the data contains only once the a point with that date. If you switch useUTC to true it behavies correctly. Also double date is displayed on certain width/number of points. I made a fixed width for this chat in order to point the issue.
